app/controllers/static_controller.rb

```diff
@@ -61,4 +62,20 @@ def set_upcoming_events
       per_page: 5 * n_events
     ).results.group_by(&:content_provider_id).map { |_p_id, p_events| p_events.first }.first(n_events)
   end
+
+  def set_catalogue_count_strings
+    catalogue_count_strings = {}
+    return catalogue_count_strings unless TeSS::Config.site.dig('home_page', 'catalogue_counts')
+
+    total_events = Event.pluck(:end).select { |x| x.present? && x >= Time.zone.now }.count
+    last_month_events = Event.pluck(:created_at).select { |x| x >= Time.zone.now - 1.month }.count
+    total_events = "#{total_events.round(-2).to_f / 1000}k" if total_events > 100
+    last_month_events = "#{last_month_events.round(-2).to_f / 1000}k" if last_month_events > 100
+    catalogue_count_strings['events'] = t('home.catalogue.events_count', total_events:, last_month_events:)
+    catalogue_count_strings['materials'] = t('home.catalogue.materials_count', total_materials: Material.all.count)
+    catalogue_count_strings['workflows'] = t('home.catalogue.workflows_count', total_workflows: Workflow.all.count)
+    catalogue_count_strings['content_providers'] = t('home.catalogue.content_providers_count', total_content_providers: ContentProvider.all.count)
+    catalogue_count_strings['trainers'] = t('home.catalogue.trainers_count', total_trainers: Trainer.all.count)
```

Comment:
I think replace these generic translations with just e.g. `pluralize(Trainer.all.count, Trainer.model_name.human)`  that way it will respect any custom translations of the TeSS resource types without the user also needing to provide all the `x_count` translations too